sw/device/tests/flash_ctrl_test.c

```diff
@@ -23,6 +23,12 @@
 #define CHECK_EQZ(x) CHECK((x) == 0)
 #define CHECK_NEZ(x) CHECK((x) != 0)
 
+#define FLASH_WORDS_PER_PAGE flash_get_words_per_page()
```

Comment:
I think it is fine to do this to get the thing up and running, but maybe before merging. All references to those defines in the code should be replaced by the function calls directly. Or what is your view on that?

Reply:
Sounds reasonable. Do you know if the DIF work already started? If yes, I would leave it as it is to not make the DIF work more cumbersome because of the required rebase. If not, I would do the replacement now to keep the DIF-related diff (what a collocation!) more readable.

Reply:
yeah i think the DIF effort is ongoing.